lib/liquid/standardfilters.rb

```diff
@@ -317,6 +317,16 @@ def default(input, default_value = "".freeze)
 
     private
 
+    def to_integer(num)
+      return num if num.is_a?(Integer)
+      num = num.to_s
+      begin
+        Integer(num)
```

Comment:
can you add a test for this line? i.e. one where the argument is a string that can be converted to an integer?